weave-js/src/components/Panel2/PanelPlot/PanelPlot.tsx

```diff
@@ -936,2851 +544,37 @@ const PanelPlotConfigInner: React.FC<PanelPlotProps> = props => {
-
-  const updateConfig2 = useCallback(
-    (change: (oldConfig: PlotConfig) => PlotConfig) => {
-      return propsUpdateConfig2!(change as any);
-    },
-    [propsUpdateConfig2]
-  );
-
-  const updateConfig2Ref = useRef<typeof updateConfig2>(updateConfig2);
-  updateConfig2Ref.current = updateConfig2;
-
-  const updateConfigRef = useRef<typeof updateConfig>(updateConfig);
-  updateConfigRef.current = updateConfig;
-
-  const concreteConfig = useMemo(
-    () => ensureValidSignals(unvalidatedConcreteConfig),
-    [unvalidatedConcreteConfig]
-  );
-
-  const configRef = useRef<PlotConfig>(config);
-  configRef.current = config;
-
-  const concreteConfigRef = useRef<ConcretePlotConfig>(concreteConfig);
-  concreteConfigRef.current = concreteConfig;
-
-  // side effect
-  useEffect(() => {
-    if (config.configVersion !== props.config?.configVersion) {
-      // persist the migrated config
-      updateConfig(config);
-    }
-  }, [config, props.config, updateConfig]);
-
-  const vegaReadyTables = useVegaReadyTables(config.series, frame);
-  const vegaCols = useMemo(
-    () =>
-      vegaReadyTables.map((vegaReadyTable, i) =>
-        PlotState.dimNames(vegaReadyTable, config.series[i].dims, weave)
-      ),
-    [config.series, vegaReadyTables, weave]
-  );
-
-  const listOfTableNodes = useMemo(() => {
-    return vegaReadyTables.map(val =>
-      opUnnest({
-        arr: TableState.tableGetResultTableNode(val, inputNode, weave)
-          .resultNode,
-      })
-    );
-  }, [vegaReadyTables, inputNode, weave]);
-
-  const flatResultNode = useMemo(() => {
-    const arrayArg: {
-      [key: number]: ReturnType<
-        typeof TableState.tableGetResultTableNode
-      >['resultNode'];
-    } = {};
-
-    const reduced = vegaReadyTables.reduce((acc, val, i) => {
-      let node: Node = listOfTableNodes[i];
-
-      if (isDash) {
-        const series = config.series[i];
-
-        const mark = getMark(series, node, series.table);
-        if (['bar', 'line', 'point', 'area'].includes(mark)) {
-          ['x' as const].forEach(axisName => {
-            node = filterTableNodeToSelection(
-              node,
-              concreteConfig.signals.domain,
-              series,
-              axisName,
-              weave.client.opStore
-            );
-          });
-        }
-        // If we have a nominal axis, limit it. We can end up with
-        // tons of unique values and Vega will basically hang.
-        const xAxisType = PlotState.getAxisType(concreteConfig.series[0], 'x');
-        const yAxisType = PlotState.getAxisType(concreteConfig.series[0], 'y');
-        if (xAxisType === 'nominal' || yAxisType === 'nominal') {
-          node = opLimit({
-            arr: node,
-            limit: constNumber(50),
-          });
-        }
-      }
-
-      /*
-      if (isDash) {
-        node = opRandomlyDownsample({
-          arr: node,
-          n: constNumber(PANELPLOT_MAX_DATAPOINTS),
-        });
-      }
-      */
```

Comment:
We should keep this in a commented out form in case we want to re-enable it later on